tiledb/bioimg/converters/base.py

```diff
@@ -190,6 +208,16 @@ def to_tiledb(
                     else:
                         group.add(os.path.basename(level_uri), relative=True)
 
+        # Register group in cloud if package exists
+        if "tiledb.cloud.groups" in sys.modules:
```

Comment:
Alternatively `if groups is not None:` (after setting `groups = None` on `ImportError` above)